gobview.opam.locked

```diff
@@ -46,13 +47,13 @@ depends: [
   "ctypes_stubs_js" {= "0.1"}
   "domain-name" {= "0.4.0"}
   "dune" {= "3.6.1"}
-  "dune-build-info" {= "3.6.1"}
+  "dune-build-info" {= "3.6.1" & dev}
```

Comment:
Why are you adding these `dev` flags here? This seems a bit strange.

Reply:
I'll retry to generate these files again such that they are consistent and also consistent with Goblint's. I think it's a bit tricky, so I probably should document it as well.